modules/src/core/ics03_connection/handler/verify.rs

```diff
@@ -56,6 +57,43 @@ pub fn verify_proofs<HostFunctions: HostFunctionsProvider>(
     }
 }
 
+#[cfg(feature = "skip_host_consensus_verification")]
+/// Entry point for verifying all proofs bundled in any ICS3 message.
+pub fn verify_proofs<HostFunctions: HostFunctionsProvider>(
```

Comment:
isn't this unused

Reply:
No, this is the function used in verify connection messages.

Reply:
oh ok